driver-scala/build.gradle

```diff
@@ -21,7 +21,7 @@ dependencies {
     compile project(path: ':bson-scala', configuration: 'default')
     compile project(path: ':driver-reactive-streams', configuration: 'default')
 
-    testImplementation project(':util')
+    testImplementation project(':util:spock')
```

Comment:
Scala uses Spock?

Reply:
No - I'll double check that nothing is marked as `@Slow`